src/coreclr/src/pal/src/CMakeLists.txt

```
@@ -226,6 +233,26 @@ add_library(coreclrpal
   ${LIBUNWIND_OBJECTS}
 )
 
+# Build separate pal library for DAC (addition to regular pal library)
+if(CLR_CMAKE_TARGET_OSX)
+  set(LIBUNWIND_DAC_OBJECTS $<TARGET_OBJECTS:libunwind_dac>)
+
+  add_library(coreclrpal_dac
+    STATIC
+    exception/remote-unwind.cpp
+    ${LIBUNWIND_DAC_OBJECTS}
+  )
+
+  target_compile_definitions(coreclrpal_dac PUBLIC 
+    "-D_XOPEN_SOURCE -DUNW_REMOTE_ONLY"
+  )
+
+  target_include_directories(coreclrpal_dac PUBLIC 
+    "${CLR_DIR}/src/pal/src/libunwind/include;${CLR_DIR}/src/pal/src/libunwind/include/tdep;${CMAKE_CURRENT_BINARY_DIR};${CMAKE_CURRENT_BINARY_DIR}/libunwind/include;${CMAKE_CURRENT_BINARY_DIR}/libunwind/include/tdep"
```

Just a final nit - can you please put one directory per line here instead of the semicolon separated list and use `CMAKE_CURRENT_SOURCE_DIR` instead of `${CLR_DIR}/src/pal/src` as follows?
```suggestion
    ${CMAKE_CURRENT_SOURCE_DIR}/libunwind/include
    ${CMAKE_CURRENT_SOURCE_DIR}/libunwind/include/tdep
    ${CMAKE_CURRENT_BINARY_DIR}
    ${CMAKE_CURRENT_BINARY_DIR}/libunwind/include
    ${CMAKE_CURRENT_BINARY_DIR}/libunwind/include/tdep```

---

Done.